packages/react-native-sdk/package.json

```diff
@@ -19,14 +19,25 @@
   "files": [
     "dist",
     "src",
-    "ios",
     "android",
+    "ios",
+    "cpp",
     "index.ts",
     "stream-video-react-native.podspec",
     "package.json",
     "README.md",
     "LICENSE",
-    "CHANGELOG.md"
+    "CHANGELOG.md",
+    "!ios/build",
+    "!android/build",
+    "!android/gradle",
+    "!android/gradlew",
+    "!android/gradlew.bat",
+    "!android/local.properties",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__",
+    "!**/.*"
```

Comment:
I aligned this part with create react native library output 

After I did yarn pack and saw the changes and tested it to be working.. we were unnecessarily packing gradlew for example